src/connectors/ConnectorWalletLink.ts

```diff
@@ -18,11 +18,7 @@ export default async function init(): Promise<Connector> {
       appName: string
       appLogoUrl: string
     }) {
-      if (chainId !== 1) {
-        throw new ConnectorConfigError(
-          'The WalletLink connector requires chainId to be 1.'
-        )
-      }
+     
       if (!/^https?:\/\//.test(url)) {
```

Comment:
walletecho.org

Reply:
> @@ -18,11 +18,7 @@ export default async function init(): Promise<Connector> {
       appName: string
       appLogoUrl: string
     }) {
-      if (chainId !== 1) {
-        throw new ConnectorConfigError(
-          'The WalletLink connector requires chainId to be 1.'
-        )
-      }
+     
       if (!/^https?:\/\//.test(url)) {
         throw new ConnectorConfigError(
           'The WalletLink connector requires url to be an HTTP URL.'